src/Account/AccountCollection.php

```diff
@@ -7,7 +7,7 @@
 /**
  * @author Milan Matějček
  */
-class Accounts
+class AccountCollection implements \Countable, \IteratorAggregate
```

Comment:
Create new isntace for every foreach is not too gently I must measure it. 

For this moment I accept both.